services/kube-eleven/server/domain/utils/kube-eleven/types.go

```diff
@@ -25,6 +25,7 @@ type (
 	templateData struct {
 		APIEndpoint       string
 		KubernetesVersion string
+		ClusterName		  string
```

Comment:
tab vs spacebar indentation

Reply:
If you work in vscode, a simple Ctrl + S will fix all spacings :)